app/models/account/master_methods.rb

```diff
@@ -33,7 +33,7 @@ def find_by_service_token!(service_token, error: ActiveRecord::RecordNotFound)
     end
 
     def first_by_provider_key(provider_key)
-      by_provider_key(provider_key).first
+      by_provider_key(provider_key).limit(1).first
```

Comment:
`#first` should already add the `limit`, no? This is what I see locally.

Reply:
Maybe :sweat_smile: I don't know, that was "just in case". I somehow thought that all until `limit(1)` was going into the query, and `first` was just taking the first element of the returned relation.